redshift_connector/cursor.py

```diff
@@ -396,7 +425,8 @@ def fetch_dataframe(self: "Cursor", num: typing.Optional[int] = None) -> typing.
         else:
             fetcheddata = self.fetchall()
 
-        result: typing.List = [tuple(column for column in rows) for rows in fetcheddata]
+        result: typing.List = [tuple(column for column in rows)
+                               for rows in fetcheddata]
```

Comment:
@Yash621 is this necessary? adds noise to the PR.

Reply:
fixed in revisions